addons/project/static/src/css/project_portal.css

```diff
@@ -0,0 +1,21 @@
+.o_project_portal_iframe {
+    background-color: #fff;
+    border: 0;
+    overflow-y: hidden;
+    width: 100%;
+}
+.o_project_portal_iframe_content {
```

Comment:
use multiple class for same properties notation
.o_project_portal_iframe {
    border: 0;
    width: 100%;
}
o_project_portal_iframe, .o_project_portal_iframe_content {
    background-color: #fff;
    overflow-y: hidden;
}